.chloggen/pgcol-add-schema-attribute.yaml

```diff
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I guess yeah, it could be a breaking change since it might invalidate some existing configurations that rely on known table names.

I do not believe a feature flag or backwards compat is required, the previous behaviour can be restored with a simple transform processor;

```yaml
transform:
  error_mode: ignore
  metric_statements:
  - context: resource
    statements:
    - set(attributes["postgresql.table.name"], Concat(attributes["postgresql.table.schema"],attributes["postgresql.table.name"],"."))
```

I can update the changelog to note it as breaking and include the suggested fix.

I don't think making previous behaviour restorable via flag or otherwise is very desirable; the current behaviour is basically broken on anything other than basic DB setups. Using multiple schemas can lead to metrics for indices being from two or more tables, and the existing table names cannot be used for things like comparing index scans to sequential scans.